templates/ovndbcluster/bin/setup.sh

```diff
@@ -87,6 +87,23 @@ if ! [ -s $DB_FILE ]; then
     cleanup_db_file
 fi
 
+# Must remove a cluster member to change protocol, replicas 1/2 will have
+# left the cluster when terminating the pod, but cannot remove final member from
+# a cluster (replica 0).
+# Convert db to standalone mode on this member instead.
+# Cluster then gets recreated by ovnctl run_*b_ovsdb using the new local address.
+if [ "$(hostname)" == "{{ .SERVICE_NAME }}-0" ]; then
+    DB_LOCAL_ADDR={{ if .TLS }}ssl{{ else }}tcp{{ end }}:$(hostname).{{ .SERVICE_NAME }}.${NAMESPACE}.svc.cluster.local:${RAFT_PORT}
+    if [ -e ${DB_FILE} ] && \
+       ovsdb-tool db-is-clustered ${DB_FILE} && \
+       [ "$(ovsdb-tool db-local-address ${DB_FILE})" != "${DB_LOCAL_ADDR}" ] \
+       ; \
+    then
+        mv -f ${DB_FILE} /tmp/ovn_db_tmp_standalone.db
```

Comment:
TODOs (input from Ihar):
try to make this atomic/re-entrant
do we need to consider disk usage when converting large dbs?
do we need to consider ready/liveness probes when converting large dbs?

Reply:
Converting a large sb db took ~12s on my nvme backed crc deployment.

The current liveness probe seems extremely trigger-happy though:
```
		TimeoutSeconds:      5,
		PeriodSeconds:       3,
		InitialDelaySeconds: 3,
```
I think it would be best to add a startup probe too.

Reply:
I think if we move conversion tasks to an init container we do not need to care about these probe timeouts

Reply:
Mention over slack with a quite large db it started in 12 seconds, so should be fine to avoid init container just for this for now.

Reply:
(For posterity, listing my understanding of this problem) the liveness check is `pidof ovsdb-server` so probably only initialization is a potential issue, and this seems to be addressed. If the liveness check ever gets more complicated, (e.g. ovsdb-server has to reply to a status command), these may be revisited.